class.jetpack.php

```diff
@@ -633,6 +638,28 @@ private function __construct() {
 		add_filter( 'jetpack_sync_before_send_updated_option', array( $this, 'jetpack_track_last_sync_callback' ), 99 );
 	}
 
+	function point_edit_links_to_calypso( $default_url, $post_id ) {
+		if ( ! $post = get_post( $post_id ) ) {
```

Comment:
Should we make our intentions more clear here? 
Are we checking for assignment or equality here?

Reply:
For assignment, but in a (unnecessarily) condensed way. I've refactored it to be more clear in 1eb5a1a454f5b1325033e4359522207d169fe527. Thanks for the suggestion!